src/ApiTestCase.php

```diff
@@ -406,8 +427,11 @@ private function getMockedResponsesFolder()
      */
     private function getCalledClassFolder()
     {
-        $calledClass = get_called_class();
-        $calledClassFolder = dirname((new \ReflectionClass($calledClass))->getFileName());
+        $calledClass       = get_called_class();
+
+        /** @var string $fileName */
+        $fileName          = (new \ReflectionClass($calledClass))->getFileName();
```

Comment:
I would prefer to not align these assignments

Reply:
Sorry, Phpstorm was that and the CI didn't catch it.